tools/depends/native/pythonmodule-setuptools/PYTHONMODULE-SETUPTOOLS-VERSION

```diff
@@ -0,0 +1,4 @@
+LIBNAME=setuptools
+VERSION=72.1.0
```

Comment:
Version 72.2.0 was released two days ago

Reply:
Dont care, im not chasing the release dragon. its been setup/tested as is, thats what im going with